config.yaml

```
@@ -2,27 +2,26 @@
 # See LICENSE file for licensing details.
 
 options:
-  bind-address-subnet:
+  bind-address:
     type: string
     description: |
-      The selected bind address will need to be part of this subnet.
-      If this is not set, it will use the IP address from the default interface.
+      Local IP address to bind for GTP and S1AP connection.
     default: ""
   enb-name:
     type: string
     description: |
       eNodeB name.
     default: dummyENB01
   enb-mcc:
-    type: int
+    type: string
     description: |
       EnodeB Mobile Country Code (MCC).
-    default: 901
+    default: '01'
```

I think that's the value for the `MNC`
```suggestion
    default: '001'
```

---

Fixed